torch/utils/tensorboard/_convert_np.py

```diff
@@ -27,7 +27,10 @@ def make_np(x):
 
 
 def _prepare_pytorch(x):
-    x = x.detach().cpu().numpy()
+    if x.dtype == torch.bfloat16:
+        x = x.to(torch.float16).detach().cpu().numpy()
```

Comment:
Instead of having an `else`, we could do:
```
if x.dtype = torch.bfloat16:
  x = x.to(torch.float16)
return x.detach().cpu().numpy()
```
Which is a bit shorter.

Reply:
Thanks. Updated the code.